src/pages/signIn/index.jsx

```diff
@@ -1,17 +1,23 @@
-import React, { useState }  from "react";
-import { useNavigate, Link } from 'react-router-dom'
-import { FiEye, FiEyeOff } from 'react-icons/fi'
+import React, { useState } from "react";
```

Comment:
You have diplicates in your import statememts. Fix it

Make sure you also import React at the top level

Reply:
Okay Chief